### ISSUE TITLE
If `params.leftArmInit` is modified, the initial configuration of the right arm is modified instead 

### ISSUE THREAD

**Issue by traversaro** [robotology/mex-wholebodymodel#39]
If `params.leftArmInit` is set (for example) to :  
~~~
params.leftArmInit  = [ -15   15  0.0  15.9   0.0]';          
~~~
Then in the visualizer the **right** arm is modified instead.

In particular, the initial configuration of the right arm appears to me modified in all the visualization, except for the one on the top right (that instead seems to be correct). 

![icubmexsim](https://cloud.githubusercontent.com/assets/1857049/10636658/6e17c4a0-7801-11e5-8dc2-45db9085a7b9.png)

**Comment by traversaro** [comment]
I reproduce this issue in the `master` branch.

**Comment by gabrielenava** [comment]
There's the same problem for the robot balancing on one foot, that should be the left foot but in the visualizer is the right one. I reported this problem in the README file and I wrote that an issue will be opened soon, but then I forgot to open it until now, sorry.

**Comment by traversaro** [comment]
No problem @gabrielenava ! I will investigate the issue then.

**Comment by gabrielenava** [comment]
fixed on branch feature/torques_linear, at #51e80d41e407938a6edc2ab86ded8cf57c57ddb2
